doc/src/manual/constructors.md

```diff
@@ -330,21 +330,22 @@ This automatic provision of constructors is equivalent to the following explicit
 julia> struct Point{T<:Real}
            x::T
            y::T
-           Point{T}(x,y) where T<:Real = new(x,y)
+           Point{T}(x,y) where {T<:Real} = new{T}(x,y)
```

Comment:
To further clarify, I feel `new(x, y)` is idiomatic here, so that's what we should show.